src/tables/sortable-table.js

```diff
@@ -47,39 +51,43 @@ const propTypes = {
   onChange: PropTypes.func,
   rowComponent: Types.component,
   headerComponent: Types.component,
-}
+  ...Caption.propTypes,
```

Comment:
I don't think this is doing anything? You aren't defining `TableCaption.propTypes` in the component, so I think this will always spread nothing.

Additionally, if you had defined `propTypes` as
```js
const propTypes = {
  className: PropTypes.string,
  caption: PropTypes.string,
}
```
then this would be overridden by the inclusion of `caption: PropTypes.string` on line 55 (and override `className: PropTypes.string` on line 44).